.ci_support/linux_64_.yaml

```diff
@@ -5,7 +5,7 @@ c_compiler:
 c_compiler_version:
 - '12'
 cdt_name:
-- cos6
+- cos7
```

Comment:
Note this is where `cdt_name` is being set, which determines the CDTs that are pulled in